migrate/loader.go

```diff
@@ -0,0 +1,137 @@
+package migrate
+
+import (
+	"fmt"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	"os"
+	"strings"
+)
+
+var loaderCommand = &cobra.Command{
+	Use:   "loader",
+	Short: "Migrate your loader versions to newer versions.",
```

Comment:
Could you specify "modloader"? Might make more sense to make this singular rather than plural as well, as the code ensures you only have 1 loader.